R/rt_interactions_type.R

```diff
@@ -1,35 +1,21 @@
-#' Retrieve all Interactions Types
+#' Retrieve Interactions Types used in TaxRef
 #'
-#' @seealso [`rt_interactions_find_type()`] to retrieve all information by
-#'          interactions
+#' If used without argument returns the list of all interactions used in TaxRef.
 #'
-#' @export
-rt_interactions_type = function() {
-
-  api_query = rt_GET("interactions/types/")
-
-  parse_taxa(api_query)
-}
-
-
-#' Retrieve a interactions Types based on its id
-#'
-#' @param interaction_id {`character(1)`}\cr{}
-#'                  A character identifying the type of interactions (see
-#'                  [`rt_interactions_type()`] for a list of interactions types)
+#' @param interaction_id {`character(1)`, default = `NULL`}\cr{}
+#'                  A character identifying the type of interactions
 #'
 #' @examples
-#' rt_interactions_type_id("PREDATEUR")
-#'
-#' @seealso see [`rt_interactions_type()`] for a list of interactions types
+#' rt_interactions_type("PREDATEUR")
```

Comment:
```suggestion
#' rt_interactions_type(interaction_id = "PREDATEUR")
```

Reply:
Does the case matter here? Other functions use `tolower()`

Reply:
For all TaxRef functions, case does matter. However as everything is uppercase here, we can use `toupper()` to make it case-insensitive.